Framework/Kernel/CMakeLists.txt

```diff
@@ -593,11 +593,15 @@ if ( MPI_BUILD )
   set ( PV_PLUGINS "" )
   set ( IGNORE_PARAVIEW "1" )
 else ()
+  option ( ENABLE_FILE_LOGGING "Enable logging to file for development builds. It is always enabled for packages" ON )
```

Comment:
Does this belong outside of the `MPI_BUILD` check? `FILELOGGER_CHANNEL` is not a defined variable if a `MPI_BUILD` is used with this version of the code.